src/styles/Navbar.scss

```diff
@@ -7,6 +7,6 @@
 }
 
 .nav-right{   
-    padding-right: 20px;
+    padding-right: 0 15px;
```

Comment:
What is that?

Reply:
Little mistake. I should say: 
padding: 0 15px;
I will change it right now

Reply:
okey